dev/src/overmind/actions.tsx

```diff
@@ -454,11 +454,13 @@ export const setActiveAssignment = ({ state }: Context, assignmentID: number): v
 }
 
 /** Rebuilds the currently active submission */
-export const rebuildSubmission = async ({ state, effects }: Context): Promise<void> => {
+export const rebuildSubmission = async ({ state, actions, effects }: Context): Promise<void> => {
     if (state.currentSubmission && state.selectedAssignment) {
         const response = await effects.grpcMan.rebuildSubmission(state.selectedAssignment.getId(), state.activeSubmission)
-        if (response.data) {
-            state.currentSubmission = response.data
+        if (success(response)) {
+            // TODO: Alerting is temporary due to the fact that the server no longer returns the updated submission.
```

Comment:
Couldn't this be fixed by simply fetching the submission using `GetSubmission` once the gRPC rebuild call returned successfully? That would at least preserve the functionality as it was.

Reply:
We actually don't have a gRPC function to fetch a single submission currently. I could add a gRPC function for this.

Reply:
Okay. Hmmm. Can we use `GetSubmissions`? We would need `CourseID` and one of `UserID` or `GroupID` at the call site. Another option could be to actually return a `Submission` from the `RebuildSubmissions` method instead of `Void`. If we are rebuilding all we could return an empty `Submission`. I prefer the first, since it would be a bit strange to return a single submission...

Reply:
Maybe it is best to leave this as is for now, and implement #625 separately to avoid that this PR becomes too big. Can you create an issue for these TODO items, so we don't forget them.